LICENSE.md

```diff
@@ -0,0 +1,116 @@
+CC0 1.0 Universal
```

Comment:
A LICENSE file was already present and this is not exactly markdown per se. Is this intentional or accidental?

Reply:
The main guidelines also suggest:

> Place a file named license or LICENSE in the repo root with the license text.

Reply:
Pardon me. I will remove the file.